reactioncommerce/reaction
# refactor: use npm package for simple-inventory plugin

Resolves https://github.com/reactioncommerce/reaction/issues/6133

This PR removes the internal `simple-inventory` plugin, and installs the new `api-plugin-inventory-simple` from npm as a replacement.

The code in that npm plugin is the same, so the expected outcome of this PR is seeing nothing different than how the app currently runs.

## Testing
- Start reaction, see that `simple-inventory` is the correct version in the `systemInformation` query
- Update the inventory for a product and see the changes

Signed-off-by: Maria Paktiti <maria.paktiti@gmail.com>
